test/integration/features/cmd-openshift.feature

```diff
@@ -5,143 +5,164 @@ cluster in VM provided by Minishift.
   Minishift openshift config view prints YAML configuration of OpenShift cluster.
   Note: --target=master is default value for minishift openshift config command
-      When executing "minishift openshift config view" succeeds
-      Then stdout should be valid YAML
+     When executing "minishift openshift config view" succeeds
+     Then stdout should be valid YAML
 
   Scenario: Seeing configuration of OpenShift node
-      When executing "minishift openshift config view --target node" succeeds
-      Then stdout should be valid YAML
+     When executing "minishift openshift config view --target node" succeeds
+     Then stdout should be valid YAML
 
   Scenario: Setting configuration on OpenShift master
-      When executing "minishift openshift config set --patch '{"assetConfig": {"logoutURL": "http://www.minishift.io"}}'" succeeds
-      Then stdout should contain "Patching OpenShift configuration"
-       And user waits "30" seconds
-      When executing "minishift openshift config view" succeeds
-      Then stdout is YAML which contains key "assetConfig.logoutURL" with value matching "http://www\.minishift\.io"
-
-  Scenario: Add a component to running openshift cluster
-      When executing "minishift openshift component add service-catalog" succeeds
-       And stdout should contain
-       """
-       Finished installing "openshift-service-catalog"
-       """
+     When executing "minishift openshift config set --patch '{"assetConfig": {"logoutURL": "http://www.minishift.io"}}'" succeeds
+     Then stdout should contain "Patching OpenShift configuration"
+      And user waits "30" seconds
+     When executing "minishift openshift config view" succeeds
+     Then stdout is YAML which contains key "assetConfig.logoutURL" with value matching "http://www\.minishift\.io"
+
+  Scenario: List out service-catalog component
+     When executing "minishift openshift component list" succeeds
+     Then stdout should contain
+      """
+      service-catalog
+      """
+
+  Scenario: Add a component to running OpenShift cluster
+     When executing "minishift openshift component add service-catalog" succeeds
+      And stdout should contain
+      """
+      Finished installing "openshift-service-catalog"
+      """
+
+  Scenario: Add a non exsisting component on a running OpenShift cluster
+     When executing "minishift openshift component add non-exsisting" fails
+     Then stderr should contain
+      """
+      use 'minishift openshift component list' to find valid components
+      """
 
   Scenario: Deleting the Minishift instance
-     Given Minishift has state "Running"
-      When executing "minishift delete --force" succeeds
-      Then Minishift should have state "Does Not Exist"
+    Given Minishift has state "Running"
+     When executing "minishift delete --force" succeeds
+     Then Minishift should have state "Does Not Exist"
+
+  Scenario: Applying component service-catalog whitout running Minishift instance
```

Comment:
here you can add tag `@quick`

Reply:
Addition of more tags resolves some problem but increases an extra layer of work for contributors for selecting the tag like ```@quick``` which i feel nobody cares. Later down the line i think it will be confusion between quick and normal scenario. 

IMO let keep it as it is and later we can think of assigning the tag if there is hard requirement. WDYT ?

Reply:
whatever

Reply:
:)

Reply:
`@quick` is part of the `make integration`, PR was approved and pulled to the project, no one complaint at that time. If nobody cares and uses it, well ok. Please go ahead and create an issue for removal of the tag from features and makefile and from documentation.

Reply:
Aha... Might be i misunderstood ```quick``` tag. What i understood is quick comprises basic + those scenario which do not require a running minishift instance.

Reply:
Exactly and I believe `Scenario: Applying component service-catalog whitout running Minishift instance` does not require a running Minishift instance. So in case of quick the first scenario and this last one will only run. Making it like 2 seconds or something like that.

BTW there is a typo in the scenario name: `whitout`.

Reply:
Done